lib/vagrant-openshift/command/download_artifacts_origin.rb

```diff
@@ -31,6 +31,10 @@ def execute
           opts = OptionParser.new do |o|
             o.banner = "Usage: vagrant download-artifacts-origin [machine-name]"
             o.separator ""
+
+            o.on("", "--include-release-artifacts", String, "Download release binaries") do |f|
```

Comment:
Having `download` as part of a flag name to a command that only does downloads seemed bad.